README.md

```diff
@@ -59,19 +59,19 @@
 
 1) Presentation
 
-   Open Source Firmware Conference 2020 - [openspdm](https://cfp.osfc.io/osfc2020/talk/ECQ88N/)

```

Comment:
This can stay openspdm since that was the name at the time.

Reply:
Yes. I have changed. Thanks.